tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932237.yaml

```diff
@@ -205,36 +205,23 @@ tests:
               Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
             method: GET
             port: 80
-            uri: /
+            uri: /get
             version: HTTP/1.0
           output:
             no_log_contains: id "932237"
   - test_title: 932237-14
+    desc: "FP with word settings"
     stages:
       - stage:
           input:
-            dest_addr: "127.0.0.1"
```

Comment:
Instead of removing this test, leave it and simply change `log_contains` to `no_log_contains`.

Reply:
is it really needed though, shouldn't the existing test be adaquate to test for FPs?

Reply:
There's no other test that tests `set` specifically, and matching of `set` is what you've changed in this PR, so no.

Reply:
hm... ok that makes sense